source/CMakeLists.txt

```diff
@@ -340,6 +345,8 @@ set_source_files_properties(
   ${CMAKE_CURRENT_SOURCE_DIR}/software_version.cpp
   PROPERTIES OBJECT_DEPENDS "${SPIRV_TOOLS_BUILD_VERSION_INC}")
 
+PCH(SPIRV_SOURCES pch.cpp)
```

Comment:
There are too many files named pch.cpp and pch.h now.  It might be best to rename them so they identify their intended use.  For example, the test/opt/ one could be called `test_opt_pch*`.  That way it will be easier to make sure we do not accidentally pick up the wrong file.

Reply:
I can give them more unique names, though it doesn't actually help with the Ninja problems (I had tried that last night).

Reply:
I've renamed all the pch files to reflect their directories.